cypress/support/nodeManager.ts

```diff
@@ -24,6 +29,37 @@ export const connectTwoNodes = (
   );
 };
 
+interface createCommitOptions {
+  repositoryPath: string;
+  radHome: string;
+  subject: string;
+  passphrase: string;
+  name?: string;
+  email?: string;
+}
+
+const credentialsHelper = (passphrase: string) =>
+  `'!f() { test "$1" = get && echo "password=${passphrase}"; }; f'`;
+
+export const createCommit = (options: createCommitOptions): void => {
+  cy.exec(
+    `set -euo pipefail
+export PATH=$PWD/target/release:$PATH
+cd ${options.repositoryPath}
+git commit --allow-empty -m "${options.subject}"
+git -c credential.helper=${credentialsHelper(options.passphrase)} push rad`,
+    {
+      env: {
+        RAD_HOME: options.radHome,
+        GIT_AUTHOR_NAME: options.name || "John McPipefail",
```

Comment:
On "John McPipefail"

Does that mean that John Doe married and change his last name?